lms/djangoapps/discussion_api/serializers.py

```diff
@@ -288,8 +297,10 @@ def validate(self, attrs):
         return attrs
 
     def restore_object(self, attrs, instance=None):
-        if instance:  # pragma: no cover
-            raise ValueError("CommentSerializer cannot be used for updates.")
+        if instance:
+            for key, val in attrs.items():
+                instance[key] = val
+            return instance
```

Comment:
instance is a Comment, which does not currently have an update function. I could probably add that; I haven't thought about whether there's a reason not to.